components/automate-ui/src/app/pages/api-token/list/api-token-list.component.html

```diff
@@ -14,6 +14,8 @@
       [conflictErrorEvent]="conflictErrorEvent"
       objectNoun="token"
       [createForm]="createTokenForm"
+      [showProjectsDropdown]="!isMajorV1 && isMinorV1"
```

Comment:
We're using `!isMajorV1 && isMinorV1` in thre places. We're doing this to exclude 1.0 and 2.0, and keeping stuff intact in case there's a 2.2 some day? hrm OK...